frontend/isoko/src/pages/Authentication/SignUp.tsx

```diff
@@ -164,28 +143,51 @@ const Description = styled.div`
    box-shadow: 5px 5px 4px rgba(0, 0, 0, 0.25);
 `;
 
+const FormError = styled.p`
+   justify-content: center;
+   color: red;
+   text-align: left;
+   width: 70%;
+   margin: 0 auto;
+`;
+
 const SignUp: React.FC = () => {
-   const [listBusinessState, setListBusinessState] = useState('');
    const [firstName, setFirstName] = useState('');
    const [lastName, setLastName] = useState('');
    const [email, setEmail] = useState('');
    const [password, setPassword] = useState('');
    const [confirmPassword, setConfirmPassword] = useState('');
+   const [err, setErr] = useState('');
 
    const navigate = useNavigate();
 
    const checkAllFieldsEntered = () => {
       if (
-         listBusinessState.length === 0 ||
          firstName.length === 0 ||
          lastName.length === 0 ||
          email.length === 0 ||
          password.length === 0 ||
          confirmPassword.length === 0
       ) {
+         setErr('Make sure all fields are filled out');
          return false;
       }
-      return true;
+      checkPasswordsMatch();
+   };
+
+   const checkPasswordsMatch = () => {
+      if (password !== confirmPassword) {
+         setErr('Make sure your passwords match');
+         return false;
+      }
+      if (password.length < 8 || confirmPassword.length < 8) {
+         // AWS cognito actually checks the password is valid, this is just to provide helpful error feedback to the users
+         setErr(
+            'Make sure your password is at least 8 digits, and contains a lowercase, uppercase, and special character'
```

Comment:
it looks like cognito has a separate error message for each issue so it doesn't show the whole password policy at once which is a little annoying. would it be too much of a hassle to add another check w a regex that looks for special characters/capital letters/etc  and show the error message that shows all the policy requirements? 
<img width="545" alt="Screen Shot 2022-01-15 at 4 34 45 PM" src="https://user-images.githubusercontent.com/43476619/149642796-ca4eaa3c-f25e-482f-8d04-44b003be10ae.png">

Reply:
ex: 

```
const regExpPassword = RegExp(
    /^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[@$!%*?&]).{8,}$/
)
```
```
isError.password = 
                    regExpPassword.test(value) ? "" : "Must be at least 8 characters long and contain at least 1 number, 1 uppercase letter, 1 lowercase letter and 1 special character.";
```

Reply:
^ that could replace the password length < 8 check

Reply:
> ^ that could replace the password length < 8 check

Good call, ill add that now